drizzlepac/hapsequencer.py

```diff
@@ -53,6 +55,67 @@
 # --------------------------------------------------------------------------------------------------------------
 
 
+def correct_hla_classic_ra_dec(orig_hla_classic_sl_name, cattype, log_level):
+    """
+    This subroutine runs Rick White's read_hla_catalog script to convert the RA and Dec values from a HLA Classic
+    sourcelist into same reference frame used by the HAP sourcelists. A new version of the input file with the
+    converted RA and Dec values is written to the current working directory named <INPUT SOURCELIST NAME>_corrected.txt.
+
+    Parameters
+    ----------
+    orig_hla_classic_sl_name : string
+        name of the HLA Classic sourcelist whose RA and Dec values will be converted.
+
+    cattype : string
+        HLA Classic catalog type. Either 'sex' (source extractor) or 'dao' (DAOphot).
+
+    log_level : int
+        The desired level of verboseness in the log statements displayed on the screen and written to the .log file.
+
+    Returns
+    -------
+    mod_sl_name : string
+        Name of the new version of the input file with the converted RA and Dec values
+    """
+    try:
+        mod_sl_name = os.path.basename(orig_hla_classic_sl_name)
+
+        # Execute read_hla_catalog.read_hla_catalog() to convert RA and Dec values
+        dataset = mod_sl_name.replace("_{}phot.txt".format(cattype), "")
+        modcat = read_hla_catalog.read_hla_catalog(dataset, cattype=cattype, applyomega=True, multiwave=False,
+                                                   verbose=True, trim=False, log_level=log_level)
+
+        # Identify RA and Dec column names in the new catalog table object
+        for ra_col_title in ["ra", "RA", "ALPHA_J2000", "alpha_j2000"]:
+            if ra_col_title in modcat.colnames:
+                true_ra_col_title = ra_col_title
+                log.debug("RA Col_name: {}".format(true_ra_col_title))
+                break
+        for dec_col_title in ["dec", "DEC", "Dec", "DELTA_J2000", "delta_j2000"]:
+            if dec_col_title in modcat.colnames:
```

Comment:
Just a comment...suggest to convert strings to lower- or upper-case before comparison.

Reply:
Resolved in commit [224e89f](https://github.com/spacetelescope/drizzlepac/pull/548/commits/224e89fe00f4431208dd872f35ec82023f9ca803)